packages/fast-components-styles-msft/src/utilities/color/neutral-focus.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { neutralForegroundRest } from "./neutral-foreground";
+import { palette, Palette, PaletteType, Swatch } from "./palette";
+import designSystemDefaults, { DesignSystem } from "../../design-system";
+import { neutralFocus } from "./neutral-focus";
+
+describe("neutralForeground", (): void => {
+    test("should return a string when invoked with an object", (): void => {
+        expect(typeof neutralFocus(designSystemDefaults)).toBe("string");
+    });
+
+    test("should return a function when invoked with a function", (): void => {
+        expect(typeof neutralFocus(() => "#FFF")).toBe("function");
+    });
+
+    test("should operate on default design system if no design system is supplied", (): void => {
+        expect(neutralFocus({} as DesignSystem)).toBe("#101010");
```

Comment:
Do we want this test to fail if we change the default design system value? Should we expect the empty object to match the same function with defaults passed?

Reply:
I think we want this to fail if we change the default design-system to something that makes this change, yea -IMO that would be a very significant change that we would want to know about if done inadvertently.